watertap/unit_models/zero_order/autothermal_hydrothermal_liquefaction_zo.py

```diff
@@ -0,0 +1,97 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of an autothermal hydrothermal liquefaction unit.
+"""
+
+from pyomo.environ import Constraint, units as pyunits, Var
+from idaes.core import declare_process_block_class
+
+from watertap.core import build_sido_reactive, constant_intensity, ZeroOrderBaseData
+
+# Some more information about this module
+__author__ = "Chenyu Wang"
+
+
+@declare_process_block_class("ATHTLZO")
+class ATHTLZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for an autothermal hydrothermal liquefaction (AT-HTL) unit.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "autothermal_hydrothermal_liquefaction"
+
+        build_sido_reactive(self)
+
+        self.flow_mass_in = Var(
+            self.flowsheet().time,
+            units=pyunits.tons / pyunits.hour,
```

Comment:
Similar to a previous comment, do you know if pyunits.tons implies metric or imperial?

Reply:
This units should be metric tons, and I believe `pyunits.tons` is metric tons.